#### DIFFERENCE SCORE (D)
$D = X_1-X_2$

#### MEAN DIFFERENCE SCORE ($\overline{D}$)

$\overline{D} = \dfrac{\Sigma{D}}{n}$

#### Statistical Hypothesis

Null Hypotheis: $H_0: \mu_D \leq 0$

Alternative/Research Hypotheis: $H_1: \mu_D > 0$

#### Two Other Possible Alternative Hypotheses
$H_0: \mu_D < 0$ translates into a one-tailed test with the lower tail critical, and a nondirectional hypothesis

$H_0: \mu_D \neq 0$ translates into a two-tailed test

#### SAMPLING DISTRIBUTION OF $\overline{D}$

$\mu_{\overline{D}}=\mu_{D}$

#### Standard error

$\sigma_{\overline{D}}=\dfrac{{\sigma}_D}{\sqrt{n}}$

#### t Test

$t = \dfrac{(sample~mean~difference)-(hypothesized~population~mean~difference)}{estimated~standard~error}$

Expressed in symbols, t RATIO FOR TWO POPULATION MEANS (TWO RELATED SAMPLES)

$t = \dfrac{\overline{D}-\mu_{D_{hyp}}}{S_\overline{D}}$

which has a t sampling distribution with n – 1 degrees of freedom. $\overline{D}$ represents the sample mean of the difference scores; $\mu_{D_{hyp}}$ represents the hypothesized population mean (of zero) for the difference scores; and $S_\overline{D}$ represents the estimated standard error of $\overline{D}$

### Sample Sums of Squares
$SS_D=\Sigma{D}^2-\dfrac{(\Sigma{D})^2}{n}$

#### SAMPLE STANDARD DEVIATION, $S_D$

$S_D = \sqrt{\dfrac{SS_D}{n-1}}$

#### ESTIMATED STANDARD ERROR, $S_\overline{D}$

$S_\overline{D}=\dfrac{S_D}{\sqrt{n}}$

#### CONFIDENCE INTERVAL FOR μD (TWO RELATED SAMPLES)

$\overline{D} \pm (t_{conf})(S_{\overline{D}})$

where $\overline{D}$ represents the sample mean of the difference scores; $t_{conf}$ represents a number (distributed with n – 1 degrees of freedom) from the t tables, which satisfies the confidence specifications; and $S_\overline{D}$ represents the estimated standard error defined in ESTIMATED STANDARD ERROR, $S_\overline{D}$

$S_\overline{D}=\dfrac{S_D}{\sqrt{n}}$

#### STANDARDIZED EFFECT SIZE, COHEN’S d (TWO RELATED SAMPLES)

$d=\dfrac{\overline{D}}{S_D}$

#### SUMMARY OF t TESTS FOR POPULATION MEANS

\begin{array}{|c|c|c|c|c|c|}
\hline
\textbf{TYPE OF
SAMPLE} & \textbf{SAMPLE
MEAN} & \textbf{NULL
HYPOTHESIS*} & \textbf{STANDARD
ERROR} & \textbf{t RATIO} & \textbf{DEGREES OF
FREEDOM} \\
\hline
One~sample & \overline{X} & H_0:\mu=some~number & S_{\overline{X}} & \dfrac{\overline{X}-\mu_{hyp}}{S_{\overline{X}}} & n-1 \\
\hline
Two~independent~samples~(no~pairing) & \overline{X}_1-\overline{X}_2 & H_0:\mu_1-\mu_2=0 & S_{\overline{X}_1-\overline{X}_2} & \dfrac{(\overline{X}_1-\overline{X}_2)-(\mu_1-\mu_2)_{hyp}}{S_{\overline{X}_1-\overline{X}_2}} & n_1+n_2-2 \\
\hline
Two~~related~samples~(pairs~of~observations) & \overline{D} & H_0:\mu_D=0 & S_{\overline{D}} & \dfrac{\overline{D}-\mu_{D_{hyp}}}{S_{\overline{D}}} & n – 1~(where~n~refers~to~pairs~of~observations) \\
\hline
\end{array}

$*$ For a two-tailed test.

#### t RATIO FOR A SINGLE POPULATION CORRELATION COEFFICIENT

$t=\dfrac{r-\rho_{hyp}}{\sqrt{\dfrac{1-r^2}{n-2}}}$

where r refers to the sample correlation coefficient, $\rho_{hyp}$ refers to the hypothesized population correlation coefficient (which always must equal zero); and n refers to the number of pairs of observations. The expression in the denominator represents the estimated standard error of the sample correlation coefficient. As implied by the term at the bottom of this expression, the sampling distribution of t has n – 2 degrees of freedom. When pairs of observations are represented as points in a scatterplot, r assumes that the cluster of points approximates a straight line. Two degrees of freedom are lost because only n – 2 points are free to vary about some straight line that, itself, always depends on two points.

#### CORRELATION COEFFICIENT (COMPUTATION FORMULA)

$r = \frac{SP_{xy}}{\sqrt{SS_xSS_y}}$

#### The statistical hypotheses must be selected from among the following three possibilities, where μD represents the population mean for all difference scores:

Non-directional:

$H_0:\mu_D=0$;
$H_0:\mu_D \neq 0$

Directional, lower tail critical:

$H_0:\mu_D \geq 0$;
$H_0:\mu_D < 0$

Directional, upper tail critical:

$H_0:\mu_D \leq 0$;
$H_0:\mu_D > 0$

The t ratio for two related samples has a sampling distribution with n – 1 degrees of freedom, given that n equals the number of paired observations. A confidence interval can be constructed for estimating μD. A single interpretation is possible only if the limits of the interval have the same sign, either both positive or both negative.

If the t test is statistically significant, Cohen’s d can be used as a standardized estimate of effect size. When using t for two related samples, you must assume that the population of difference scores is normally distributed. You need not be too concerned about violations of this assumption as long as sample sizes are relatively large. To test the hypothesis that the population correlation coefficient equals zero, use a new t ratio with n – 2 degrees of freedom.

#### SUMMARY OF t TESTS FOR POPULATION MEANS

\begin{array}{|c|c|c|c|c|c|}
\hline
\textbf{TYPE OF
SAMPLE} & \textbf{SAMPLE
MEAN} & \textbf{NULL
HYPOTHESIS*} & \textbf{STANDARD
ERROR} & \textbf{t RATIO} & \textbf{DEGREES OF
FREEDOM} \\
\hline
One~sample & \overline{X} & H_0:\mu=some~number & S_{\overline{X}} & \dfrac{\overline{X}-\mu_{hyp}}{S_{\overline{X}}} & n-1 \\
\hline
Two~independent~samples~(no~pairing) & \overline{X}_1-\overline{X}_2 & H_0:\mu_1-\mu_2=0 & S_{\overline{X}_1-\overline{X}_2} & \dfrac{(\overline{X}_1-\overline{X}_2)-(\mu_1-\mu_2)_{hyp}}{S_{\overline{X}_1-\overline{X}_2}} & n_1+n_2-2 \\
\hline
Two~~related~samples~(pairs~of~observations) & \overline{D} & H_0:\mu_D=0 & S_{\overline{D}} & \dfrac{\overline{D}-\mu_{D_{hyp}}}{S_{\overline{D}}} & n – 1~(where~n~refers~to~pairs~of~observations) \\
\hline
\end{array}

$*$ For a two-tailed test.

In [1]:
import statistics
import math

X1 = [12,5,11,11,9,18]
X2 = [7,3,4,6,3,13]

X1_mean = statistics.mean(X1)
X2_mean = statistics.mean(X2)
print(f'X1_mean = {X1_mean}\nX2_mean = {X2_mean}')

D = [X1[n]-X2[n] for n in range(0,len(X1))]
print(D)
D_mean = statistics.mean(D)
print(f'D_mean = {D_mean}')

X1_mean = 11
X2_mean = 6
[5, 2, 7, 5, 6, 5]
D_mean = 5


#### Function for t Test for repeated measures

In [122]:
# CALCULATIONS FOR THE t TEST: REPEATED MEASURES (EPO EXPERIMENT)
X1 = [12,5,11,11,9,18]
X2 = [7,3,4,6,3,13]
def t_test_repeated_measures(a,b):
    import statistics
    import math

    n = len(a)
    X1_mean = statistics.mean(X1)
    X2_mean = statistics.mean(X2)
    D_overline = X1_mean - X2_mean
    print(f'D_overline = {D_overline}')
    
    D = [X1[n]-X2[n] for n in range(0,len(X1))]
    # print(f'D = {D}')

    sum_D = sum(D)
    print(f'sum_D = {sum_D}')

    sq_of_each_D = [d**2 for d in D]
    print(f'sq_of_each_D = {sq_of_each_D}')

    sum_of_sq_of_each_D = sum(sq_of_each_D)
    print(f'sum_of_sq_of_each_D = {sum_of_sq_of_each_D}')

    SSD = sum_of_sq_of_each_D - (sum_D**2/n)
    print(f'SSD = {SSD}')

    sample_standard_deviation_SD = math.sqrt(SSD/(n-1))
    print(f'sample_standard_deviation_SD = {sample_standard_deviation_SD}')

    est_std_error = sample_standard_deviation_SD/math.sqrt(n)
    print(f'est_std_error = {est_std_error}')

    t_ratio = (D_overline-0)/est_std_error
    print(f't_ratio = {t_ratio}')

    d = D_overline/sample_standard_deviation_SD
    print(f"Cohen's d = {d}")
t_test_repeated_measures(X1,X2)

D_overline = 5
sum_D = 30
sq_of_each_D = [25, 4, 49, 25, 36, 25]
sum_of_sq_of_each_D = 164
SSD = 14.0
sample_standard_deviation_SD = 1.6733200530681511
est_std_error = 0.6831300510639733
t_ratio = 7.319250547113998
Cohen's d = 2.988071523335984


#### Markdown for CALCULATIONS FOR THE t TEST: REPEATED MEASURES (EPO EXPERIMENT)
$\overline{D}=5$

$\Sigma{D}=30$

$\Sigma{D^2}=164$

$SS_D=14.0$

$S_D=1.6733200530681511$

$S_\overline{D}=0.6831300510639733$

$t=7.319250547113998$

#### Progress Check *15.2 An investigator tests a claim that vitamin C reduces the severity of common colds. To eliminate the variability due to different family environments, pairs of children from the same family are randomly assigned to either a treatment group that receives vitamin C or a control group that receives fake vitamin C. Each child estimates, on a 10-point scale, the severity of their colds during the school year. The following scores are obtained for ten pairs of children:

\begin{array}{ccc}
\text{PAIR NUMBER} & \text{VITAMIN C (X1)} & \text{FAKE VITAMIN C (X2)} \\
\hline
1 & 2 & 3 \\
2 & 5 & 4 \\
3 & 7 & 9 \\
4 & 0 & 3 \\
5 & 3 & 5 \\
6 & 7 & 7 \\
7 & 4 & 6 \\
8 & 5 & 8 \\
9 & 1 & 2 \\
10 & 3 & 5 \\
\end{array}

Using t, test the null hypothesis at the .05 level of significance.

#### Progress Check 15.3 Referring to the vitamin C experiment in Question 15.2, construct and interpret a 95 percent confidence interval for the population mean difference score.

#### Progress Check *15.4 For the vitamin C experiment in Question 15.2, estimate and interpret the standardized effect size, d, given a mean, $\overline{D}$, of –1.50 days and a standard deviation, $S_D$, of 1.27 days.

In [121]:
X1 = [2,5,7,0,3,7,4,5,1,3]
X2 = [3,4,9,3,5,7,6,8,2,5]
# X1 = [12, 5, 11, 11, 9, 18]
# X2 = [7, 3, 4, 6, 3, 13]
def t_test_repeated_measures(a,b):
    import statistics
    import math

    n = len(a)
    X1_mean = statistics.mean(X1)
    X2_mean = statistics.mean(X2)
    D_overline = X1_mean - X2_mean
    print(f'D_overline = {D_overline}')
    
    D = [X1[n]-X2[n] for n in range(0,len(X1))]
    # print(f'D = {D}')

    sum_D = sum(D)
    print(f'sum_D = {sum_D}')

    sq_of_each_D = [d**2 for d in D]
    # print(f'sq_of_each_D = {sq_of_each_D}')

    sum_of_sq_of_each_D = sum(sq_of_each_D)
    print(f'sum_of_sq_of_each_D = {sum_of_sq_of_each_D}')

    SSD = sum_of_sq_of_each_D - (sum_D**2/n)
    print(f'SSD = {SSD}')

    sample_standard_deviation_SD = math.sqrt(SSD/(n-1))
    print(f'sample_standard_deviation_SD = {sample_standard_deviation_SD}')

    est_std_error = sample_standard_deviation_SD/math.sqrt(n)
    print(f'est_std_error = {est_std_error}')

    t_ratio = (D_overline-0)/est_std_error
    print(f't_ratio = {t_ratio}')

    d = D_overline/sample_standard_deviation_SD
    print(f"Cohen's d = {d}")

    # At 95% confidence level, significance level of 0.05, df=10-1=9, two-tailed test, t_value=2.262
    CI1 = D_overline - (2.262*est_std_error)
    CI2 = D_overline + (2.262*est_std_error)
    print(f'CI1 = {CI1}\nCI2 = {CI2}')
t_test_repeated_measures(X1,X2)

D_overline = -1.5
sum_D = -15
sum_of_sq_of_each_D = 37
SSD = 14.5
sample_standard_deviation_SD = 1.2692955176439846
est_std_error = 0.40138648595974313
t_ratio = -3.7370465934182993
Cohen's d = -1.181757895737503
CI1 = -2.4079362312409387
CI2 = -0.5920637687590611


#### Markdown Progress Check *15.2

### Statistical Hypothesis

Null Hypotheis: $H_0: \mu_D \leq 0$

$\overline{D}=-1.5$

$\Sigma{D}=-15$

$\Sigma{D^2}=37$

$SS_D=14.5$

$S_D=1.2692955176439846$

$S_\overline{D}=0.40138648595974313$

$t=-3.7370465934182993$

$d=-1.181757895737503$

At 95% confidence, significance level of 0.05, df=9, one-tailed test, t_value=±1.833, since t_ratio=-3.7370465934182993, is more negative than t_value, reject the null hypothesis

Note: ChatGPT uses a two-tailed test, which has a t_value=±2.262, at df=10-1=9, both reject the null hypothesis


CI1 = -2.4079362312409387

CI2 = -0.5920637687590611

When schoolchildren are matched for home environment, we are 95 percent confident that the interval between –0.60 and –2.40 covers the reduction in estimated severity of colds.

#### Progress Check *15.5 Each of the following studies requires a t test for one or more population means. Specify whether the appropriate t test is for one sample, two independent samples, or two related samples, and in the last case, whether it involves repeated measures or matched pairs of different subjects.

(a) College students are randomly assigned to receive either behavioral or cognitive therapy. After twenty therapeutic sessions, each student earns a score on a mental health questionnaire. Answer: t test for two independent samples

(b) A researcher wishes to determine whether attendance at a day-care center increases the scores of three-year-old children on a motor skill test. Random assignment dictates which twin from each pair of twenty twins attends the day-care center and which twin stays at home. (Such a draconian experiment doubtless would incur great resistance from the parents, not to mention the twins!) Answer: t test for two related samples, matched pairs

(c) One hundred college freshmen are randomly assigned to sophomore roommates who have either similar or dissimilar vocational goals. At the end of their first year, the mean GPAs of these two groups are to be analyzed. Answer: t test for two independent samples

(d) According to the U.S. Department of Health, the average 16-year-old male can do 23 push-ups. A physical education instructor finds that in his school district, 30 randomly selected 16-year-old males can do an average of 28 pushups. Answer: t test for one sample

(e) A child psychologist assigns aggression scores to each of 10 children during two 60-minute observation periods separated by an intervening exposure to a series of violent TV cartoons. Answer: t test for two related samples, repeated measures

#### Progress Check *15.6 A random sample of 27 California taxpayers reveals an r of .43 between years of education and annual income. Use t to test the null hypothesis at the .05 level of significance that there is no relationship between educational level and annual income for the population of California taxpayers.

#### t RATIO FOR A SINGLE POPULATION CORRELATION COEFFICIENT

$t=\dfrac{r-\rho_{hyp}}{\sqrt{\dfrac{1-r^2}{n-2}}}$

In [4]:
# 95% confidence, at 0.05 significance level
import math

n = 27
r = 0.43

t = (r - 0)/(math.sqrt((1-r**2)/(n-2)))
t

2.3814036412717288

#### 15.7 An educational psychologist wants to check the claim that regular physical exercise improves academic achievement. To control for academic aptitude, pairs of college students with similar GPAs are randomly assigned to either a treatment group that attends daily exercise classes or a control group. At the end of the experiment, the following GPAs are reported for the seven pairs of participants:

\begin{array}{ccc}
\text{PAIR NUMBER} & \text{PHYSICAL EXERCISE (X1)} & \text{NO PHYSICAL EXERCISE (X2)} \\
\hline
1 & 4.00 & 3.75 \\
2 & 2.67 & 2.74 \\
3 & 3.65 & 3.42 \\
4 & 2.11 & 1.67 \\
5 & 3.21 & 3.00 \\
6 & 3.60 & 3.25 \\
7 & 2.80 & 2.65 \\
\end{array}

(a) Using t, test the null hypothesis at the .01 level of significance.

(b) Specify the p-value for this test result. Answer p < 0.05, from the table with df=6, in between 2.447 and 3.707, two-tailed test

(c) If appropriate (because the test result is statistically significant), use Cohen’s d to estimate the effect size.

(d) How might this test result be reported in the literature?

#### DIFFERENCE SCORE (D)
$D = X_1-X_2$

#### MEAN DIFFERENCE SCORE ($\overline{D}$)

$\overline{D} = \dfrac{\Sigma{D}}{n}$

#### Statistical Hypothesis

Null Hypotheis: $H_0: \mu_D \leq 0$

Alternative/Research Hypotheis: $H_1: \mu_D > 0$

#### Two Other Possible Alternative Hypotheses
$H_0: \mu_D < 0$ translates into a one-tailed test with the lower tail critical, and a nondirectional hypothesis

$H_0: \mu_D \neq 0$ translates into a two-tailed test

#### SAMPLING DISTRIBUTION OF $\overline{D}$

$\mu_{\overline{D}}=\mu_{D}$

#### Standard error

$\sigma_{\overline{D}}=\dfrac{{\sigma}_D}{\sqrt{n}}$

#### t Test

$t = \dfrac{(sample~mean~difference)-(hypothesized~population~mean~difference)}{estimated~standard~error}$

Expressed in symbols, t RATIO FOR TWO POPULATION MEANS (TWO RELATED SAMPLES)

$t = \dfrac{\overline{D}-\mu_{D_{hyp}}}{S_\overline{D}}$

which has a t sampling distribution with n – 1 degrees of freedom. $\overline{D}$ represents the sample mean of the difference scores; $\mu_{D_{hyp}}$ represents the hypothesized population mean (of zero) for the difference scores; and $S_\overline{D}$ represents the estimated standard error of $\overline{D}$

### Sample Sums of Squares
$SS_D=\Sigma{D}^2-\dfrac{(\Sigma{D})^2}{n}$

#### SAMPLE STANDARD DEVIATION, $S_D$

$S_D = \sqrt{\dfrac{SS_D}{n-1}}$

#### ESTIMATED STANDARD ERROR, $S_\overline{D}$

$S_\overline{D}=\dfrac{S_D}{\sqrt{n}}$

#### CONFIDENCE INTERVAL FOR μD (TWO RELATED SAMPLES)

$\overline{D} \pm (t_{conf})(S_{\overline{D}})$

where $\overline{D}$ represents the sample mean of the difference scores; $t_{conf}$ represents a number (distributed with n – 1 degrees of freedom) from the t tables, which satisfies the confidence specifications; and $S_\overline{D}$ represents the estimated standard error defined in ESTIMATED STANDARD ERROR, $S_\overline{D}$

$S_\overline{D}=\dfrac{S_D}{\sqrt{n}}$

#### STANDARDIZED EFFECT SIZE, COHEN’S d (TWO RELATED SAMPLES)

$d=\dfrac{\overline{D}}{S_D}$

#### Markdown for CALCULATIONS FOR THE t TEST: REPEATED MEASURES (EPO EXPERIMENT)

$\Sigma{D}=1.5599999999999996$

$\Sigma{D^2}=0.503$

$\overline{D}=0.2228571428571428$

$SS_D=14.0$

$S_D=1.6733200530681511$

$S_\overline{D}=0.6831300510639733$

$t=7.319250547113998$

In [124]:
# a) Using t, test the null hypothesis at the .01 level of significance, df=7-1=6, t_value=±3.707
X1 = [4.00,2.67,3.65,2.11,3.21,3.60,2.80]
X2 = [3.75,2.74,3.42,1.67,3.00,3.25,2.65]
# X1 = [12, 5, 11, 11, 9, 18]
# X2 = [7, 3, 4, 6, 3, 13]
# X1 = [2,5,7,0,3,7,4,5,1,3]
# X2 = [3,4,9,3,5,7,6,8,2,5]
def t_test_repeated_measures(a,b):
    import statistics
    import math

    n = len(a)
    X1_mean = statistics.mean(X1)
    X2_mean = statistics.mean(X2)
    D_overline = X1_mean - X2_mean
    print(f'D_overline = {D_overline}')
    
    D = [X1[n]-X2[n] for n in range(0,len(X1))]
    # print(f'D = {D}')

    sum_D = sum(D)
    print(f'sum_D = {sum_D}')

    sq_of_each_D = [d**2 for d in D]
    # print(f'sq_of_each_D = {sq_of_each_D}')

    sum_of_sq_of_each_D = sum(sq_of_each_D)
    print(f'sum_of_sq_of_each_D = {sum_of_sq_of_each_D}')

    SSD = sum_of_sq_of_each_D - (sum_D**2/n)
    print(f'SSD = {SSD}')

    sample_standard_deviation_SD = math.sqrt(SSD/(n-1))
    print(f'sample_standard_deviation_SD = {sample_standard_deviation_SD}')

    est_std_error = sample_standard_deviation_SD/math.sqrt(n)
    print(f'est_std_error = {est_std_error}')

    t_ratio = (D_overline-0)/est_std_error
    print(f't_ratio = {t_ratio}')

    d = D_overline/sample_standard_deviation_SD
    print(f"Cohen's d = {d}")
t_test_repeated_measures(X1,X2)

# t_ratio = 3.66442265284744 is less than critical t_value = 3.707, we retain the null hypothesis

D_overline = 0.22285714285714286
sum_D = 1.5599999999999996
sum_of_sq_of_each_D = 0.503
SSD = 0.15534285714285728
sample_standard_deviation_SD = 0.16090517763725384
est_std_error = 0.06081644067012074
t_ratio = 3.664422652847441
Cohen's d = 1.3850215768665575


#### STANDARDIZED EFFECT SIZE, COHEN’S d (TWO RELATED SAMPLES)

$d=\dfrac{\overline{D}}{S_D}$